src/components/organisms/Gravy.js

```diff
@@ -0,0 +1,73 @@
+import React, {useState} from "react";
+import styled from "styled-components";
+import {graphql, useStaticQuery} from "gatsby";
+import Gallery from "../molecules/Gallery";
+
+const Holder = styled.div`
+`;
+
+const Inner = styled.div`
+  padding: 15px;
+  display: grid;
+  grid-template-columns: 1fr;
+  align-content: start;
+  grid-gap: 24px;
+  @media (${(props) => props.theme.breakpoints.md}) {
+    padding: 24px;
+  }
+
+  > div {
+    > :first-child {
+      margin-top: 0;
+    }
+
+    > :last-child {
+      margin-bottom: 0;
+    }
+  }
+`;
+
+const GalleryHolder = styled.div`
+  border-top: 1px solid;
+`;
+
+function Gravy(props) {
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const data = useStaticQuery(graphql`
+      query GravyQuery {
+          prismicGravy {
+              data {
+                  gallery {
+                    caption {
+                        text
```

Comment:
Can you fill in the captions on Prismic? We are missing them on the deploy preview.

Reply:
There is only a caption on the fist image on the live site.